api/src/main/java/org/openmrs/Person.java

```diff
@@ -325,6 +327,21 @@ public void setCauseOfDeath(Concept causeOfDeath) {
 		this.causeOfDeath = causeOfDeath;
 	}
 	
+	/**
+	 * @return cause of person's death non coded
+	 * @since Trunk-5405
```

Comment:
since takes on the version number when this is going to be released

Reply:
@dkayiwa thanks for the review!

I have no idea when it should be released. There is also no detail provided in the issue description as far as I see.

Do you know where I can determine the release number? Or do you know when it will be released?